blocks/header/header.js

```diff
@@ -192,85 +161,51 @@ function decorateMiddleNav(nav) {
   a.setAttribute('aria-label', 'Sunstar Home');
 }
 
-/* Decorate the other items - which is the items pulled from top nav */
-function decorateOtherItems(otherItemsEl) {
-  otherItemsEl.classList.add('other-items');
-
-  /* Pull items from the top nav */
-  document.querySelector('nav.nav-top').querySelectorAll(':scope>ul>li').forEach((li) => {
-    otherItemsEl.appendChild(li.cloneNode(true));
-  });
-
-  /* Make a lang picker for mobile */
-  const langPicker = document.createElement('li');
-  langPicker.classList.add('mobile-lang-picker');
-  const langPickerUl = document.createElement('ul');
-  langPicker.appendChild(langPickerUl);
-  otherItemsEl.querySelector('.lang-picker').querySelectorAll(':scope>ul>li').forEach((li) => {
-    langPickerUl.appendChild(li.cloneNode(true));
-  });
-
-  langPicker.querySelectorAll(':scope>ul>li').forEach((li) => {
-    li.classList.add('mobile-lang-picker-item');
-    li.classList.remove('lang-picker-item');
+function getNavbarToggler() {
+  const navbarToggl = htmlToElement(`<button class="navbar-toggler">
+  <span class="mobile-icon">
+    <i></i>
+    <i></i>
+    <i></i>
+    <i></i>
+  </span>
+  </button>`);
+  navbarToggl.addEventListener('click', () => {
+    const navBottom = document.querySelector('.nav-bottom');
+    const header = document.querySelector('header');
+    const { body } = document;
+    if (navBottom.classList.contains('open')) {
+      navBottom.classList.remove('open');
+      header.classList.remove('menu-open');
+      body.style.position = '';
+    } else {
+      navBottom.classList.add('open');
+      header.classList.add('menu-open');
+      body.style.position = 'fixed';
+    }
+    window.addEventListener('resize', () => {
```

Comment:
I think `resize` event may have [performance implications](https://tigeroakes.com/posts/resize-event-alternatives/). Can you please check? If yes, I'd simply omit this event.

Reply:
used js based media query